lib/std/os.zig

```diff
@@ -3190,3 +3190,33 @@ pub fn sched_yield() SchedYieldError!void {
         else => return error.SystemCannotYield,
     }
 }
+
+// TODO support for non-null terminated strings?
```

Comment:
null terminated strings are now available! would you mind rebasing and using them?

Reply:
I have the feeling I found a bug with sentinel-terminated pointers in `zig fmt`... :thinking: 

```
$ zig fmt lib/std/os.zig
lib/std/os.zig:3215:53: error: Expected '{', found 'IntegerLiteral'
fn toMemFdPath(name: []const u8) ![MFD_MAX_NAME_LEN:0]u8 {
                                                    ~
```

Reply:
after https://github.com/ziglang/zig/issues/3771#issuecomment-558276712 I feel the changes I did were the supposed ones (still getting the hang of it)

Reply:
You can remove this TODO now.